tests/test_google_ads_discovery.py

```diff
@@ -12,12 +12,10 @@ class DiscoveryTest(GoogleAdsBase):
     def expected_fields(self):
```

Comment:
We should get rid of this method entirely. It's not being used anwywhere

Reply:
Removed